pkg/clusterapi/extraargs.go

```diff
@@ -51,7 +52,22 @@ func PodIAMAuthExtraArgs(podIAMConfig *v1alpha1.PodIAMConfig) ExtraArgs {
 	return args
 }
 
-func ResolvConfExtraArgs(resolvConf *v1alpha1.ResolvConf) ExtraArgs {
+func NodeCidrMaskExtraArgs(clusterNetwork *v1alpha1.ClusterNetwork) ExtraArgs {
+	if clusterNetwork == nil {
+		return nil
+	}
+	if clusterNetwork.Nodes == nil {
+		return nil
+	}
+	if clusterNetwork.Nodes.CidrMaskSize == nil {
+		return nil
+	}
+	args := ExtraArgs{}
+	args.AddIfNotEmpty("node-cidr-mask-size", strconv.Itoa(*clusterNetwork.Nodes.CidrMaskSize))
+	return args
+}
+
+func ResolveConfExtraArgs(resolvConf *v1alpha1.ResolvConf) ExtraArgs {
```

Comment:
Why changing from `resolv` to `resolve`?
This is a specific name :) https://man7.org/linux/man-pages/man5/resolv.conf.5.html

Reply:
my editor kept giving me a warning so i changed it, I'll put it back